dev-packages/e2e-tests/test-applications/vue-3/src/views/ComponentMainView.vue

```diff
@@ -0,0 +1,10 @@
+<script setup lang="ts">
+import ComponentOneView from './ComponentOneView.vue';
+import ComponentTwoView from './ComponentTwoView.vue';
+</script>
+
+<template>
+  <h1>Demonstrating Component Tracking</h1>
+  <ComponentOneView />
+  <ComponentTwoView />
+</template>
```

Comment:
Inspired by the [sveltekit-2-svelte-5](https://github.com/getsentry/sentry-javascript/blob/8a8f08f789be5effff39a3643f0eb9319b842c37/dev-packages/e2e-tests/test-applications/sveltekit-2-svelte-5/src/routes/components/%2Bpage.svelte#L1-L15) test setup.